sandpack-react/src/contexts/sandpackContext.tsx

```diff
@@ -338,7 +366,6 @@ class SandpackProvider extends React.PureComponent<
     if (client) {
       client.cleanup();
       delete this.clients[clientId];
-      delete this.unsubscribeQueuedListeners[clientId];
```

Comment:
This was causing the main error when unmounting a client. So, I think we can remove this at all since unsubscription functions need to handle the entire list before cleaning themselves from this list.

Reply:
Yes, this is a bit over-optimized, the important thing is that the unsubscribe functions get called. `this.unsubscribeQueuedListeners[clientId]` would not be used after this if block executes

Reply:
Yeah, I wouldn't worry about this extra optimization, because the object will be empty at the end of unsubscribing listener process.